### ISSUE TITLE
[Wii] GB, GBC code dump upon load

### ISSUE THREAD

**Issue by SuperrSonic** [mgba-emu/mgba#516]
Tested nightly 1bb496d

**Comment by endrift** [comment]
wtf, I KEEP FIXING this same bug

**Comment by GB-CG** [comment]
Didn't happen to me. What I do if this happens is deleting the config file, retry, and it should be fine.

**Comment by endrift** [comment]
Never delete the config. Move it out of the way and attach it if that fixes the problem. Otherwise, I can't debug something I can't reproduce.

**Comment by SuperrSonic** [comment]
I tested a fresh config and it did work, exited and returned it still worked. But once I changed screen mode to stretched GB games crashed, tried again but with filtering saved to bilinear and GBA games crashed.

I noticed that the default filtering says it is pixelated but when you start a fresh config it's bilinear instead.
Also tried frameskip set to 2, it seemed to work but when I load the game again it code dumps, no matter the GB type.
